youtube_dl/extractor/seznamzpravy.py

```diff
@@ -0,0 +1,128 @@
+        'info_dict': {
+            'id': '38489',
+            'ext': 'mp4',
+            'title': 'ČSSD a ANO nás s\xa0elektronickou evidencí podrazily, říká šéf lidovců',
+            'description': 'Předvolební rozhovory s lídry deseti hlavních stran pokračují. Ve Výzvě Jindřicha Šídla odpovídal předseda lidovců Pavel Bělobrádek.',
+        }
+    }]
+
+    def _extract_sdn_formats(self, sdn_url, video_id):
+        sdn_data = self._download_json(sdn_url, video_id)
+        formats = []
+        for fmt, fmtdata in sdn_data.get('data', {}).get('mp4', {}).items():
+            relative_url = fmtdata.get('url')
+            if not relative_url:
+                continue
+
+            try:
+                width, height = fmtdata.get('resolution')
+            except (TypeError, ValueError):
+                width, height = None, None
+
+            formats.append({
+                'format_id': fmt,
+                'width': int_or_none(width),
+                'height': int_or_none(height),
+                'url': urljoin(sdn_url, relative_url),
+            })
+
+        playlists = sdn_data.get('pls', {})
+        dash_rel_url = playlists.get('dash', {}).get('url')
+        if dash_rel_url:
+            formats.extend(self._extract_mpd_formats(urljoin(sdn_url, dash_rel_url), video_id, mpd_id='dash', fatal=False))
+
+        hls_rel_url = playlists.get('hls', {}).get('url')
+        if hls_rel_url:
+            formats.extend(self._extract_m3u8_formats(urljoin(sdn_url, hls_rel_url), video_id, ext='mp4', m3u8_id='hls', fatal=False))
+
+        self._sort_formats(formats)
+        return formats
+
+    def _extract_caption(self, api_data, video_id):
+        title = api_data.get('title') or api_data.get('captionTitle')
+        caption = api_data.get('caption')
+        if not title or not caption:
+            return {}
+
+        if 'sdn' in caption.get('video', {}):
+            sdn_url = caption['video']['sdn'] + self._MAGIC_SUFFIX
+        elif 'liveStreamUrl' in caption:
+            sdn_url = self._download_json(caption['liveStreamUrl'] + self._MAGIC_SUFFIX, video_id)['Location']
+        else:
+            return {}
+
+        formats = self._extract_sdn_formats(sdn_url, video_id)
+        if formats:
+            return {
+                'id': video_id,
+                'title': title,
+                'description': api_data.get('perex'),
+                'display_id': api_data.get('slug'),
+                'formats': formats,
+            }
+
+    def _extract_content(self, api_data, video_id):
+        entries = []
+        for num, item in enumerate(api_data.get('content', [])):
+            media = item.get('properties', {}).get('media', {})
+            sdn_url_part = media.get('video', {}).get('sdn')
+            title = media.get('title')
+            if not sdn_url_part or not title:
+                continue
+
+            entry_id = '%s-%s' % (video_id, num)
```

Comment:
That's not an id. Id is something that identifies a video not article.
The whole current extraction approach is incorrect. What should be done is a separate extractor for iframe URL from
```
<iframe src="https://www.seznam.cz/zpravy/iframe/player?duration=241&serviceSlug=zpravy&src=https%3A%2F%2Fv39-a.sdn.szn.cz%2Fv_39%2Fvmd%2F5999c902ea707c67d8e267a9%3Ffl%3Dmdk%2C432f65a0%7C&itemType=video&autoPlay=false&title=Sv%C4%9Bt%20bez%20obalu%3A%20%C4%8Ce%C5%A1t%C3%AD%20voj%C3%A1ci%20na%20mis%C3%ADch%20(kr%C3%A1tk%C3%A1%20verze)&series=Sv%C4%9Bt%20bez%20obalu&serviceName=Seznam%20Zpr%C3%A1vy&poster=%2F%2Fd39-a.sdn.szn.cz%2Fd_39%2Fc_img_F_I%2FR5puJ.jpeg%3Ffl%3Dcro%2C0%2C0%2C1920%2C1080%7Cres%2C1200%2C%2C1%7Cjpg%2C80%2C%2C1&width=1920&height=1080&cutFrom=0&cutTo=0&splVersion=VOD&contentId=170889&contextId=35990&showAdvert=true&collocation=&autoplayPossible=true&embed=&isVideoTooShortForPreroll=false&isVideoTooLongForPostroll=true&videoCommentOpKey=&videoCommentId=&version=4.0.76&dotService=zpravy&gemiusPrismIdentifier=bVc1ZIb_Qax4W2v5xOPGpMeCP31kFfrTzj0SqPTLh_b.Z7&zoneIdPreroll=seznam.pack.videospot&skipOffsetPreroll=5&sectionPrefixPreroll=%2Fzpravy"
```
and an article extractor that finds iframes and delegates to iframe extractor (or an addition to generic extractor instead if such videos can be embedded on any site).